docs/dataformats/observation_lists.rst

```diff
@@ -22,23 +22,48 @@ This has the advantage that everyone can work with whatever tool they like. Here
 * `csvkit <https://csvkit.readthedocs.org/en/latest/>`_
 * Your own script using e.g the `Python standard library CSV module <http://docs.python.org/2/library/csv.html>`_ or `pandas <http://pandas.pydata.org>`_
 
-A run list must have at least a column called ``Run``::
+A run list must have at least a column called ``OBS_ID``::
  
-   Run
+   OBS_ID
    42
    43
 
-Usually it has many more columns with information about each run::
- 
-   Run,Telescope_Pattern,Date,Duration,GLON,GLAT,Target,Zenith
-   1234,24,2013-03-22 14:32,1832,83.7,-5.2,Crab Nebula,32
+Usually it has many more columns with information about each observation. A list of Gammapy supported columns is::
+
+   OBS_ID: observation ID as an integer (starting at 0 or 1? does it matter?)
```

Comment:
Can you please re-format this as an RST simple table?
See [here](http://sphinx-doc.org/rest.html#tables) or [here](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables) for a description of the format and [here](http://astropy.readthedocs.org/en/latest/time/index.html) for an example of usage in the Astropy docs. Note that on any Astropy Sphinx docs page you can click "Page Source" at the very bottom and see the RST file ... this is sometimes easier to learn how to do something than to read the RST or Sphinx docs.